pkgs/desktops/gnome-3/core/gnome-tour/default.nix

```diff
@@ -49,6 +49,7 @@ rustPlatform.buildRustPackage rec {
   ];
 
   # Don't use buildRustPackage phases, only use it for rust deps setup
+  dontUseCargoBuild = true;
```

Comment:
gnome-tour was already fixed in #114673 and the approach in that PR is better, these packages should just move away from `buildRustPackage` and use the hook.